src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -197,6 +220,10 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
             />
           )}
         </Responsive>
+        <ErrorModal
+          onClose={this.onCloseModal.bind(this)}
```

Comment:
Given that `onCloseModal` is defined as an arrow function I don't think that you have to bind it to the instance... right? (I'm not 100% sure about that, but that's how I thought it worked).